jetty-distribution/src/main/resources/docs/index.html

```diff
@@ -0,0 +1,2 @@
+It works!!
```

Comment:
This file should not be added to the distro.   Instead the module should list the docroot directory as a needed directory and create it as empty - only if the module is enabled.    We could copy in a template index.html from `modules/fileserver/index.html`